extensions/funqy/funqy-server-common/deployment/src/main/java/io/quarkus/funqy/deployment/FunctionScannerBuildStep.java

```diff
@@ -167,10 +167,11 @@ public FunctionInitializedBuildItem staticInit(FunctionRecorder recorder, List<F
         recorder.init();
         for (FunctionBuildItem function : functions) {
             if (function.getFunctionName() == null) {
-                recorder.register(context.classProxy(function.getClassName()), function.getMethodName());
+                recorder.register(context.classProxy(function.getClassName()), function.getMethodName(),
```

Comment:
I noticed that `classProxy()` has been deprecated should we change it in future?